pkg/apis/messaging/v1beta1/channel_defaults.go

```diff
@@ -19,7 +19,7 @@ package v1beta1
 import (
 	"context"
 
-	"knative.dev/eventing/pkg/logging"
+	"github.com/google/knative-gcp/pkg/logging"
```

Comment:
same for all of this... 

I'm almost sure there is a ""knative.dev/pkg/logging"

Reply:
disregard, @grantr mentioned the different signatures